style.css

```diff
@@ -834,17 +1763,186 @@ body {
     width: 21px;
     height: 26px;
   }
-}
 
-@media (max-width: 880px) {
-  .wrap__img,
-  .wrap__main-img,
-  .wrap__article,
+  .second-page {
+    min-height: 620px;
+  }
+
+  .second-page__img {
+    margin-right: 69px;
+  }
+
+  .second__img_purses {
+    width: 87%;
+    padding: 76px 0 0 94px;
+  }
+
+  .second__wrap {
+    margin: 36px 28px 0 0;
+  }
+
+  .second-page__article {
+    width: 255px;
+    padding: 28px 44px 30px 37px;
+    margin-bottom: 22px;
+  }
+
+  .second-page__title {
+    font-size: 28px;
+  }
+
+  .second-page__subtitle {
+    font-size: 11px;
+    letter-spacing: .7px;
+    margin-bottom: 40px;
+  }
+
+  .second-page__description {
+    font: 600 11px/15px 'Montserrat-Regular', sans-serif;
+    margin-bottom: 24px;
+    letter-spacing: -.4px;
+  }
+
+  .second-page__color-options {
+    margin-bottom: 29px;
+  }
+
+  .eclipse {
+    width: 22px;
+    height: 22px;
+    margin-right: 9px;
+  }
+
+  .second-page__price {
+    margin-bottom: 38px;
+    font-size: 28px;
+    margin-right: 7px;
+  }
+
+  .second-page__quantity {
+    padding: 4px;
+  }
+
+  .second__minus {
+    padding: 2px 7px;
+  }
+
+  .second__quantity__input {
+    width: 5px;
+    padding: 2px;
+  }
+
+  .second__plus {
+    padding: 2px 7px;
+  }
+
+  .second__add-to-cart {
+    font-size: 9px;
+    width: 175px;
+  }
+
+  .second__more-items {
+    font-size: 9px;
+  }
+
+  .second__more-vector {
+    width: 46%;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #ebe3d8 267px, #000 267px, #000 318px, #ebe3d8 318px);
+    min-height: 620px;
+  }
+
+  .third-page__img {
+    padding-left: 61px;
+  }
+
+  .third-page__img-clothes {
+    width: 569px;
+  } 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)